generators/chipyard/src/main/scala/config/RocketConfigs.scala

```diff
@@ -90,24 +90,20 @@ class MulticlockRocketConfig extends Config(
   new freechips.rocketchip.subsystem.WithNBigCores(1) ++
   // Frequency specifications
   new chipyard.config.WithTileFrequency(1600.0) ++       // Matches the maximum frequency of U540
-  new chipyard.config.WithSystemBusFrequency(800.0) ++   // Ditto
+  new chipyard.clocking.WithClockGroupsCombinedByName(("uncore"   , Seq("sbus", "cbus", "implicit")),
+                                                      ("periphery", Seq("pbus", "fbus"))) ++
+  new chipyard.config.WithSystemBusFrequency(800.0) ++   // Matches the maximum frequency of U540
   new chipyard.config.WithMemoryBusFrequency(1000.0) ++  // 2x the U540 freq (appropriate for a 128b Mbus)
-  new chipyard.config.WithPeripheryBusFrequency(100) ++  // Retains the default pbus frequency
-  new chipyard.config.WithSystemBusFrequencyAsDefault ++ // All unspecified clock frequencies, notably the implicit clock, will use the sbus freq (800 MHz)
+  new chipyard.config.WithPeripheryBusFrequency(100) ++  // Slow periphery bus
```

Comment:
Skimming that, I see "The recommended frequency of coreclk is 1.0 GHz" and "The L2 cache and peripherals such as UART, SPI, I2C, and PWM operate in a single clock domain (tlclk) running at coreclk/2 rate.". This can be updated to match that.